cmd/neofs-node/netmap.go

```diff
@@ -315,10 +315,13 @@ func (c *cfg) netmapLocalNodeState(epoch uint64) (*netmapSDK.NodeInfo, error) {
 	nmNodes := nm.Nodes()
 	for i := range nmNodes {
 		if bytes.Equal(nmNodes[i].PublicKey(), c.binPublicKey) {
+			c.localNodeInNetmap.Store(true)
```

Comment:
Why not doing it in `c.handleLocalNodeInfo()`?

Reply:
BTW, how about officially offline (on a map, but offline) nodes? Will they be able to replicate?

Reply:
Offline nodes don't reside network map. What do you mean by "able to replicate"? Anyone is able to replicate.

Reply:
> Why not doing it in `c.handleLocalNodeInfo()`?

Sure, missed the place.

Reply:
> Anyone is able to replicate.

Nodes that are not a part of the map can't replicate (well, they can try, but their objects are not accepted by other nodes), so my question is can other nodes accept objects from nodes in "offline" state?

Reply:
There is no special logic for offline nodes.